### ISSUE TITLE
Fails when no cache

### ISSUE THREAD

**Issue by Aashishkebab** [d4rken/sdmaid-public#3624]
![Screenshot_20200624-133240](https://user-images.githubusercontent.com/25588487/85604437-4bca1e00-b61f-11ea-8804-6a7441901a66.png)

It tried clearing the cache for an app that has no cache. It got stuck there, likely because it was unable to detect the button pressed event or new screen event or however it is doing it via accessibility.

**Comment by d4rken** [comment]
What size did SD Maid display before?

**Comment by Aashishkebab** [comment]
I'm not sure what it displayed "in the app", but that was in settings after
it was going through in accessibility mode and clicking on each app's
"Clear Cache" button.

Best regards,
Aashish Bharadwaj

**Comment by d4rken** [comment]
I have checks in place that filter out empty apps.
It's possible that a race-condition with the system leads to it still trying empty apps, but it shouldn't happen every time.

Does it every happen every time?

How were you able to start SD Maid on an app that has no cache?
